install.php

```diff
@@ -0,0 +1,109 @@
+<?php
+// Verify that we are allowed to start installing
+
+if(file_exists('./application/config/database.php'))
+{
+	exit("It seems that you have already installed this package. If you mean to re-install, please delete 'database.php' located in './application/config/'");
+}
+
+echo '<link href="./assets/css/bootstrap.min.css" rel="stylesheet" media="screen">';
+echo '<link href="./assets/css/main.css" rel="stylesheet">';
+echo '<title>Store Control Panel Setup</title>';
+
+
+if (isset($_POST['submit']))
+{
+	processInfo();
+} else {
+	requestInfo();
+}
+
+function requestInfo()
+{
+	// This is the way novices will be installing the script.
+	// That is why I only ask them for critical details, the rest can remain default.
+	echo <<<HTML
+		<div class="container">
+			<h1>Setup</h1>
+			<p>In order for this package to function, you must supply access details to a MySQL database, and a secret encryption key (this can be random).</p>
```

Comment:
You should also note, that this database needs to contain the tables for the store plugin